apps/storefront/docs/patterns/spacings/spacings.mdx

```diff
@@ -4,3 +4,35 @@ metaTitle: Spacings
 mode: draft
 route: /patterns/spacings/
 ---
+
+**Using the right spacings throughout you design will provide consistency to your product**
```

Comment:
Re-write: 
Using the ~~right~~correct spacing~~s~~ throughout you**r** design will provide consistency to your product**.**

Reply:
Updated